packages/core/admin/admin/src/pages/SettingsPage/components/SettingsNav/index.js

```diff
@@ -35,14 +38,24 @@ const SettingsNav = ({ menu }) => {
     defaultMessage: 'Settings',
   });
 
+  const handleClickOnLink = (destination = null) => {
```

Comment:
just curious why you set a value here? If nothing is passed won't it just be null / undefined?

Reply:
Not really sure why it's required either, I just thought I'd copy the [exact implementation](https://github.com/strapi/strapi/blob/430461f493de4210c3d73f42f7d09a5d34eb3d24/packages/core/admin/admin/src/components/LeftMenu/index.js#L98-L101) of the LeftMenu. Maybe it makes a difference in Amplitude whether it's null or undefined? 🤷

Reply:
10-4 works for me 🎈